src/Core/Workspace/Workspace.cs

```diff
@@ -77,6 +77,11 @@ public string CacheFolder
         /// </summary>
         public string Root { get; set; }
 
+        /// <summary>
+        /// Gets the source files to be built for this Workspace.
+        /// </summary>
+        public IEnumerable<string> SourceFiles => Directory.EnumerateFiles(Root, "*.qs", SearchOption.TopDirectoryOnly);
```

Comment:
Ordinarily I'd make this private, but I'm making it public here because I'll be using it in a subsequent PR.